test.sh

```diff
@@ -41,14 +47,18 @@ dlHttpFiles() {
 }
 
 dlTftpFiles() {
-  dir=${WORKSPACE}/build/static/tftp
-  if [ "${REPO_NAME}" != "on-tftp" ]; then
+  if [ -n "${MULTI}" ]; then
+    dir=${WORKSPACE}/on-tftp/static/tftp
+  else
+    dir=${WORKSPACE}/build/static/tftp
+  fi
+  if [[ "${REPO_NAME}" != *"on-tftp"* ]]; then
       dir=${WORKSPACE}/build-deps/on-tftp/static/tftp
   fi
   mkdir -p ${dir} && cd ${dir}
   # pull down index from bintray repo and parse files from index
   wget --no-check-certificate https://dl.bintray.com/rackhd/binary/ipxe/ && \
-      exec grep 'href=' index.html | awk '{print $3}' | cut -c8-200 | sed -e "s/\"//g" > files
+      exec grep 'href=' index.html | awk '{print $2}' | cut -d \< -f 1 | cut -d \> -f 2 > files
```

Comment:
:+1:   great job , @iceiilin @PengTian0 @changev  to fix this and quick root cause!!
minor comment, it's more safe to use regex to implement.
example:
``` cat index.html |grep -o href=.*\"|sed 's/href=//' | sed 's/"//g' ```